spec.html

```diff
@@ -185,7 +189,7 @@ <h2 class="nojs">Specification Version 1.1</h2>
                   <tr>
                     <td class="fix nt"></td>
                     <td class="fix op">|</td>
-                    <td class="fix ex">"\x11" e_name int64</td>
+                    <td class="fix ex">"\x11" e_name uint64</td>
```

Comment:
Thanks for the review, but I can't merge this PR. Not sure who has permission to do so.